src/webapp/src/client/app/common/directives/wall/message-show-more/message-show-more.directive.js

```diff
@@ -1,86 +1,133 @@
 (function () {
-    'use strict';
+    "use strict";
```

Comment:
Isn't it single quotes should be a default? Maybe in Simoona project it's not specifically configured, but usually it's a standard default.
If such things are not clear - for open-source projects usual practice is to leave such things alone and not change them.